3DCalendar/ViewController.swift

```diff
@@ -14,6 +14,26 @@ class ViewController: UIViewController, ARSCNViewDelegate {
 
     @IBOutlet var sceneView: ARSCNView!
     
+    let colors = [UIColor.red, UIColor.orange, UIColor.yellow, UIColor.green, UIColor.blue, UIColor.purple, UIColor.magenta, UIColor.cyan, UIColor.black, UIColor.gray, UIColor.orange, UIColor.white]
+
+
+    let calendarViewControllers: [UIViewController] = {
```

Comment:
Since we're gonna start allowing the `CalendarViewController` to be created with a `Date` We can rewrite the property to look like this:

```
let components = Calendar.current.dateComponents(in: .current, from: Date())
let range = Calendar.current.range(of: .month, in: .year, for: Date())!

let months = range.compactMap {
    DateComponents(calendar: components.calendar,
                   timeZone: components.timeZone,
                   year: components.year,
                   month: $0,
                   day: 1).date
}
return months.map { CalendarViewController.make(date: $0) }
```

Reply:
This has the advantage of being internationalized out of the box, so it'll work everywhere in the world!